people/EladTal14.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "אלעד טל",
+  "github": "EladTal14",
+  "facebook": "https://www.facebook.com/elad.tal",
+  "twitter": "https://twitter.com/EladTal4",
+  "description": "מפתח front-end חדש, גיימר אבל גם אוהב כדורעף וספורט.",
+  "linkedin": "https://www.linkedin.com/in/elad-tal-0355531b7/",
+  "customImage": true
+}
```

Comment:
@EladTal14 
Please open separate commit for adding card.
its not related to the task

Reply:
Ok, no problem